doc/rst/usingchapel/building.rst

```diff
@@ -62,132 +56,149 @@ display its help message using:
+  is in your path.
 
-Currently supported platforms include 32- and 64-bit Linux, Mac OS X,
-Cygwin (Windows), SunOS, a variety of current HPE and Cray platforms, and a
-few systems by other vendors.  Most UNIX-based environments ought to
-support Chapel (subject to the assumptions in :ref:`readme-prereqs`), but may
-not be supported "out-of-the-box" by our current Makefile structure.
-See the section below on platform-specific settings for more
-information on adding support for additional UNIX-compatible
-environments.
+* by installing Chapel to a specific location—perhaps one that is
+  already in your path—using the instructions in the next section.
 
-Note that a single Chapel installation can simultaneously support
-Chapel for multiple platforms and compiler options because all
-platform-specific binary files and executables are stored in
-subdirectories named by ``CHPL_`` environment variables.
+
+.. _readme-installing:
+
+-----------------
+Installing Chapel
+-----------------
+
+Chapel can be built and installed to a specific location as follows:
+
+.. code-block:: bash
+
+  ./configure  # use ``./configure --help`` for specific options
+  make
+  make install
+
+Running ``./configure`` will save your current configuration of
+``$CHPL_`` settings into a ``chplconfig`` file.  Running it with the
+``--prefix`` or ``chpl-home`` options permits you to specify where and
+how Chapel should be installed during the ``make install`` step.
+Specifically:
+
+* ``--prefix=/dir/for/install/`` causes the Chapel compiler,
+  libraries, and supporting code to be installed into the directories:
+
+  - ``/dir/for/install/bin``
+  - ``/dir/for/install/lib``
+  - ``/dir/for/install/share``
+
+  This technique is designed to install Chapel using a standard
+  directory structure for the purposes of integrating it into a
+  standard location that is already be in your path, such as
+  ``/usr/local/`` or ``~/``.  Note that elevated privileges are likely
+  to be required for any system-wide installation locations.
+
+* ``--chpl-home=/dir/for/install`` copies key files and directories
+  from the Chapel source tree into ``/dir/for/install``, preserving
+  Chapel's traditional directory structure.
+
+
+-----------------------------------------
+Switching Between Multiple Configurations
+-----------------------------------------
+
+A single installation of Chapel can simultaneously support multiple
+configuration options, such as target platforms or compilers, because
+all binary files and executables are stored in unique subdirectories
+as determined by the ``CHPL_`` environment variables.  However, note
+that each ``make`` command only builds the compiler and runtime for
+the current set of ``CHPL_`` environment variables defined by—and
+inferred for—your environment.  To build support for additional
+configurations, you will need to modify your ``CHPL_`` environment
+variables and re-make.
```

Comment:
So if I built different configurations in different locations, how would I switch between them?

Reply:
Good point.  I added a second paragraph to address that:

> Once you have built multiple configurations, you can switch between
> them either by setting your ``CHPL_`` environment variables to the
> same values as when the configuration was built, or using the ``chpl``
> compiler's command-line options that correspond to the environment
> variables (see the 'Compiler Configuration Options' section of the
> :ref:`man-chpl` man page for details).